src/streaming/utils/CapabilitiesFilter.js

```diff
@@ -246,11 +246,16 @@ function CapabilitiesFilter() {
         if (settings.get().streaming.capabilities.filterVideoColorimetryEssentialProperties) {
             Object.assign(config, _convertHDRColorimetryToConfig(rep));
         }
-        let supportedFlag = config.isSupported;
+        let colorimetrySupported = config.isSupported;
+        
         if (settings.get().streaming.capabilities.filterHDRMetadataFormatEssentialProperties) {
             Object.assign(config, _convertHDRMetadataFormatToConfig(rep));
         }
-        if (!supportedFlag) config.isSupported == false; // restore this flag as it may got overridden by 2nd Object.assign
+        let metadataFormatSupported = config.isSupported;
+
+        if (!colorimetrySupported && !metadataFormatSupported) {
```

Comment:
Does this need to be changed to:

````js
if (!colorimetrySupported || !metadataFormatSupported) {
  config.isSupported = false; 
}
````

Reply:
Oh,... yes, of course.
Hmm, this indicates that there's room for improving unit tests...

Reply:
I think `config.isSupported == false` still needs to be changed to `config.isSupported = false`